internal/backend/triggers/triggers.go

```diff
@@ -28,20 +31,75 @@ func (m *triggers) Create(ctx context.Context, trigger sdktypes.Trigger) (sdktyp
 
 	trigger = trigger.WithNewID()
 
+	if trigger.ConnectionID() == fixtures.BuiltinSchedulerConnectionID {
+		if schedule, _ := trigger.Data()[sdktypes.ScheduleExpression].ToString(); schedule != "" {
```

Comment:
validate in trigger that schedule is string

Reply:
actually it's already like this
- if non-empty but invalid schedule passed, then temporal will complain and no trigger will be created/updated
- if none schedule is passed we cannot distinguish between cron trigger and normal trigger, so it will complain about not having connection (since both schedule and connection will be empty in proto), so this is a valid error

Reply:
comment that the session will complain if we crash and we need to monitor it

Reply:
added a NOTE in the code